deploy.sh

```diff
@@ -1,41 +1,77 @@
 #!/usr/bin/env bash
+set -Eeuo pipefail
+
+
```

Comment:
In some cases you use singular and in some double breaklines and I presume you did it on purpose. If it matters for some reason maybe it would be worth to leave //some kind of docstring?

Reply:
it's just a visual separation of bits in script file. Most of the actions in the script are doubled (one for frontend and then right after one for backend, or one for api lambda and right after one for migration lambda) - so the double newline is just to visually help recognize particular 'steps' in script.